src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -4,6 +4,10 @@ Release History
 ===============
 * Minor fixes: Get default location from resource group when not provided for creating servers and add validation for retention days. 
 
+0.3.8
```

Comment:
The entry above for the new release should be good enough, so you can remove yours